src/Queue/CachedTransformer.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace eLife\Bus\Queue;
+
+use Doctrine\Common\Cache\Cache;
+use eLife\ApiSdk\ApiSdk;
+use eLife\ApiSdk\Model\Model;
+use Psr\Log\LoggerInterface;
+
```

Comment:
could you add a docblock here that explains the difference between `get()` and `refresh()` and the use cases from which they are called?